src/erlfmt.erl

```diff
@@ -467,8 +467,8 @@ nodes_in_range(Nodes, StartLocation, EndLocation) ->
 
 node_intersects_range(Node, StartLocation, EndLocation) ->
     {Start, End} = get_location_range(Node),
-    ((Start < StartLocation) and (End >= StartLocation)) or
-        ((Start >= StartLocation) and (Start =< EndLocation)).
+    ((Start < StartLocation) and (End > StartLocation)) or
```

Comment:
Just interested to know, why was this change required?

Reply:
This was an intermediate fix for my previous attempt, to fix one of the format_range test cases. Without trimming the unparseable ranges could be side by side (ie the end location of first one can be the same as the start location of the following). Now that the raw_strings are trimmed and the end_locations are matching this cannot happen, but I thought it still makes sense to make this function theoretically correct.